google/iree
# Set IREE codegen to Linalg on tensors by default.

- Delete outdated tests, e.g., iree/test/e2e/hackability/* and
  iree/test/e2e/structural/*
- Test LinalgToSPIRV lit tests with old pipeline. They should be fixed
  at some point, tracking in https://github.com/google/iree/issues/5155.
- Disable LocallyConnected1D integration tests. Mahesh has some clues on
  it and it is just a single test case. We don't make it be blocker,
  tracking in https://github.com/google/iree/issues/5310
- Disable conv 1d and 3d tests, see https://github.com/google/iree/issues/5150
- Add a flag to use legacy pipeline for VMLA, it should be removed once
  VMLA is deprecated or the legacy flow pipeline is deleted. This avoids
  fixing many tests regarding to VMLA.

This depends on
https://github.com/llvm/llvm-project/commit/f4eb681dc37ae84e08579bf96cd2a6f58c44f260
and
https://github.com/llvm/llvm-project/commit/c3614358452e5050b5b191fd3df3fad8b2664221
and
https://github.com/google/iree/pull/5374

It is blocked by https://github.com/google/iree/issues/5372

Fixes https://github.com/google/iree/issues/5043
